app/assets/stylesheets/components/_navbar.scss

```diff
@@ -1,3 +1,7 @@
+.btn-primary {
```

Comment:
btn-primary is a bootstrap class already. Can you give this a unique name and then apply the new class in the relevant views, instead of btn-primary?